src/pal/src/misc/dactableaddress.cpp

```diff
@@ -0,0 +1,156 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+//
+
+/*++
+
+
+
+Module Name:
+
+    dactableaddress.cpp
+
+Abstract:
+
+    Functions to write and read DAC global pointer table address between the coreclr
+    and DAC/debugger processes.
+
+Revision History:
+
+
+--*/
+
+#include "pal/palinternal.h"
+#include <sys/types.h>
+#include <unistd.h>
+#include <stdio.h>
+
+/*++
+Function
+    PAL_PublishDacTableAddress
+
+Parameters
+
+    address
+       [in] address of dac table
+    size
+       [in] size of dac table
+
+Return Values
+       pal errors
+
+--*/
+PALIMPORT
+DWORD
+PALAPI
+PAL_PublishDacTableAddress(
+    IN PVOID address,
+    IN ULONG size)
+{
+    DWORD ret = NO_ERROR;
+
+    // TODO - 3/5/15 - the DAC side needs the debuggee pid
+    // pid_t pid = getpid();
+    pid_t pid = 0;
+    char fileName[100]; 
+    snprintf(fileName, sizeof(fileName), "/tmp/%d_dacTable", pid);
```

Comment:
I would be interested to know why you chose this design, instead of generating the table at build time - how it is done on Windows, and how it used to be done for Silverlight.